engines/groovie/groovie.cpp

```diff
@@ -307,26 +306,28 @@ Common::Error GroovieEngine::run() {
 			// Still waiting for input, just update the mouse, game timer and then wait a bit more
 			_grvCursorMan->animate();
 			_system->updateScreen();
-			tmr++;
+
 			// Wait a little bit between increments.  While mouse is moving, this triggers
 			// only negligably slower.
-			if (tmr > 4) {
+			if (tmr >= 250) {
```

Comment:
do we know how often this timer is actually supposed to tick? I think it was too fast before because in the T7G knights puzzle Stauf would say "don't take all knight" like every move for casual players